requirements.txt

```diff
@@ -8,8 +8,9 @@ languagecodes==1.1.1
 countrytagger==0.1.2
 pyicu==2.12
 google-cloud-vision==3.7.2
-tesserocr==2.6.2
+tesserocr==2.7.1
 spacy==3.6.1
+numpy<2.0.0 # pinned because otherwise spacy requires an incompatible numpy
```

Comment:
For future reference because I was curious what this is about: https://github.com/explosion/spaCy/issues/13528

Seems like the issue is fixed in the latest release, so as soon as we upgrade spacy to 3.8.2 or later, this isn’t needed anymore. (Was able to confirm that the build succeeds with that version.)

Reply:
I would try to update spacy here and now, thanks!